migrations/container/20210902112747_site-command_install_iproute2.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace EE\Migration;
+
+use EE;
+use EE\Migration\Base;
+use EE\RevertableStepProcessor;
+use EE\Model\Site;
+
+class InstallIproute2 extends Base {
+
+	private $sites;
+	/** @var RevertableStepProcessor $rsp Keeps track of migration state. Reverts on error */
+	private static $rsp;
+
+	/**
+	 * Execute php config updates.
+	 *
+	 * @throws EE\ExitException
+	 */
+	public function up() {
+
+		if ( $this->skip_this_migration ) {
+			EE::debug( 'Skipping add-ext-config-8.0 update migration as it is not needed.' );
+
+			return;
+		}
+
+		$os = EE::launch( 'lsb_release -i | awk \'{print $3}\'' )->stdout;
```

Comment:
`lsb_release` may not be present everywhere. For example, this will directly fail in MacOS. Please use, `php_uname( 's' )`.